src/main/java/frc/robot/RobotState.java

```diff
@@ -502,4 +509,9 @@ public synchronized Pose2d getVisionTargetToGoalOffset() {
 
   // not sure we need it!
   public synchronized void outputToSmartDashboard() {}
+
```

Comment:
probably don't need to synchronize the getter, only the mutator

Reply:
Gotcha